src/test/java/core/basesyntax/service/impl/ActivityStrategyImplTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.service.impl;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+import core.basesyntax.model.FruitActivity;
+import core.basesyntax.service.activity.strategy.ActivityHandler;
+import core.basesyntax.service.activity.strategy.BalanceHandler;
+import core.basesyntax.service.activity.strategy.PurchaseHandler;
+import core.basesyntax.service.activity.strategy.ReturnHandler;
+import core.basesyntax.service.activity.strategy.SupplyHandler;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class ActivityStrategyImplTest {
+    private static final Map<FruitActivity.Type, ActivityHandler> ACTIVITY_HANDLER_MAP
+            = new HashMap<>();
+
+    static {
+        ACTIVITY_HANDLER_MAP.put(FruitActivity.Type.BALANCE, new BalanceHandler());
+        ACTIVITY_HANDLER_MAP.put(FruitActivity.Type.SUPPLY, new SupplyHandler());
+        ACTIVITY_HANDLER_MAP.put(FruitActivity.Type.PURCHASE, new PurchaseHandler());
+        ACTIVITY_HANDLER_MAP.put(FruitActivity.Type.RETURN, new ReturnHandler());
+    }
```

Comment:
use annotation `beforeAll` to setup sth before all tests 
<img width="986" alt="image" src="https://github.com/mate-academy/jv-fruit-shop-tests/assets/32339836/e1ff9a39-b83f-48ed-acef-c8e2fd41ac16">